tutorial/output-cache-open-ai/OutputCacheDallESample/Program.cs

```diff
@@ -0,0 +1,47 @@
+using Microsoft.AspNetCore.OutputCaching;
+using OutputCacheDallESample;
+using Redis.OM;
+using Redis.OM.Vectorizers;
+
+var builder = WebApplication.CreateBuilder(args);
+
+// add services
+builder.Services.AddStackExchangeRedisOutputCache(options => {
+    options.Configuration = builder.Configuration["RedisCacheConnection"];
+});
+builder.Services.AddOutputCache(options => {
+    // optional: named output-cache profiles
+});
+
+var app = builder.Build();
+
+app.MapGet("/", async (HttpContext context) => { await context.Response.WriteAsync("<h1>Welcome to OpenAI Art Gallery</h1>"); });
```

Comment:
Semicolons and curly brackets aren't needed for single-line `Func`s. Also please format all `MapGet()` calls consistently

Reply:
fixed. will include in the next commit.